### PR TITLE
fix(suspensive.org): add fallback for response with 202 status code

### LINKED ISSUE #1051: [Bug]: GitHub fetching don't work. Fetch succeeds only intermittently.
### Bug description

https://suspensive.org/docs/contributors

image

@saul-atomrigs Could I assign you this issue?

### Expected behavior

_No response_

### To Reproduce

_No response_

### Possible Solution

_No response_

### etc.

_No response_

### PR DESCRIPTION
**Pull request by manudeli**
close #1051 

# Overview

@saul-atomrigs I added fallback original image for GitHub api response with 202, 403 status code
And I use @suspensive/react, @suspensive/react-query to refactor your code.

Jul-09-2024 03-46-29

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: docs/suspensive.org/src/components/BubbleChart.tsx [hunk 1]
```diff
@@ -1,6 +1,115 @@
...
+        <br />
+        <img
+          src="https://contrib.rocks/image?repo=toss/suspensive"
+          alt="contributors"
+          data-canonical-src="https://contrib.rocks/image?repo=toss/suspensive"
+          style={{ maxWidth: '100%' }}
+        />
+      </Link>
+    }
+  >
```
Comment (manudeli) [inline docs/suspensive.org/src/components/BubbleChart.tsx:63]: fallback for 403, or any other not res.ok!

File: docs/suspensive.org/src/components/BubbleChart.tsx [hunk 2]
```diff
@@ -1,6 +1,115 @@
...
+            <Link href="https://github.com/toss/suspensive/graphs/contributors">
+              <br />
+              <img
+                src="https://contrib.rocks/image?repo=toss/suspensive"
+                alt="contributors"
+                data-canonical-src="https://contrib.rocks/image?repo=toss/suspensive"
+                style={{ maxWidth: '100%' }}
+              />
+            </Link>
+          )
```
Comment (manudeli) [inline docs/suspensive.org/src/components/BubbleChart.tsx:107]: fallback for res.ok but [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)!

**Review by manudeli — COMMENTED** [review]
@saul-atomrigs Check this Pull Request out please

**Comment by saul-atomrigs** [comment]
@manudeli Thaaanks!! Very nice on adding the fallback UI and the suspensive libraries 🙏🏻

**Pull request merged**